TileRarities.lua

```diff
@@ -0,0 +1,131 @@
+--[[
+    The data for the different chances and multiplier values can be found on Data:Tile_Chances
+]]--
+local p = {}
+
+function p.main(frame)
+	return p._main(frame.args)
+end
+
+function p._main(args)
+	local json = mw.loadJsonData('Data:Tile_Chances')
+    local ranks = json.ranks
+	local dom = mw.html.create()
+	local tbl = dom:tag('table'):addClass('wikitable'):css('text-align', 'right')
+	local row = tbl:tag('tr')
+	row:tag('th'):attr('rowspan', 2):wikitext('Rarity'):css('vertical-align', 'bottom')
+    for i, v in ipairs(ranks) do
+        row:tag('th'):attr('colspan', 3):wikitext(v.display)
+    end
+    row = tbl:tag('tr')
+    for i, v in ipairs(ranks) do
```

Comment:
put a `_` as the variable name if you arent actually using that value

so this can be, `for _, _ in ipairs(ranks) do`